client/src/components/date/dashboardCalendar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import DayPicker from 'react-day-picker';
+
+const DashboardCalendar = ({ subscriptions }) => {
+  const dueDates = [];
+  subscriptions.map((subscription) => {
+    const { dueDate } = subscription;
+    dueDates.push(new Date(dueDate.nextDueDate));
+    return 'hi';
+  });
```

Comment:
actually, if you're just tryna get an array of dueDates and want to use `.map()` you don't need to create and array and push into it. instead you should just do:

```suggestion
  const dueDates = subscriptions.map(({ dueDate }) => new Date(dueDate.nextDueDate));
```